setup.py

```diff
@@ -26,7 +26,9 @@
         'GitPython',
         'jinja2',
         'jsonschema',
-        'PyInquirer',
+        # 'PyInquirer>1.0.3',
+        # Need the new release of PyInquirer, see nf_core/launch.py for details
+        'PyInquirer @ git+ssh://git@github.com/CITGuru/PyInquirer.git@master#egg=PyInquirer',
```

Comment:
Just like you I am working with PyInquirer and need the latest release.

Do you think that the solution here is suitable for PyPI?
https://github.com/pypa/pip/issues/4187#issuecomment-415067034

Or are you for now just hoping that there will be a new release soonish?

Reply:
All right.

So I will go with https://github.com/tmbo/questionary . 
Seems to be much more up to date (tech wise) and more actively maintained.
Looks like it supports pretty much the same stuff as PyInquirer.

Might be of interest to you?

Reply:
Yes I've looked over questionary, there was another one mentioned in the threads too. I wasn't too keen on rewriting the code for a different library though, especially when the two features I needed are already merged into PyInquirer. I'm hoping that he does a new minor release soon. If we get close to doing a new release of nf-core/tools and it's still not out then I'll come back to this and look into questionary.

It's kind of ironic, as apparently PyInquirer was forked from another previous project because of the lack of updates 😅